packages/dom/test/unit/lib/fixtures/withAttributesTestComponent.tsx

```diff
@@ -0,0 +1,9 @@
+import { ReactElement } from "react";
+
+export function WithAttributesTestComponent(): ReactElement {
+    return (
+      <button role="button" type="submit" className="btn primary" disabled>
```

Comment:
Always use values on attributes (I'll add an ESLint rule later 🙂 )
```suggestion
      <button role="button" type="submit" className="btn primary" disabled={true}>
```

Reply:
Thanks @JoseLion, I have addressed this [here](https://github.com/stackbuilders/assertive-ts/pull/132/commits/229b78ed9e43e637f0b4c8f76929f913d611ae16).